### PR TITLE
Preserve existing pool payout_instructions when creating a PoolWallet

### PR DESCRIPTION
**Pull request by paninaro**
When syncing a wallet db from scratch, if a pool nft launcher is found, we were previously overwriting the pool config's `payout_instructions` with a new puzzle hash. This change will preserve the existing `payout_instruction` puzzle hash, if found in the config.

Issue #10478

### CODE REVIEW

**Comment by paninaro** [comment]
If this approach looks good, I'll add a couple of tests to cover the change.

File: chia/pools/pool_wallet.py [hunk 1]
```diff
@@ -229,16 +229,17 @@ async def get_unconfirmed_transactions(self) -> List[TransactionRecord]:
...
         pool_config_dict: Dict[bytes32, PoolWalletConfig] = {c.launcher_id: c for c in pool_config_list}
         existing_config: Optional[PoolWalletConfig] = pool_config_dict.get(current_state.launcher_id, None)
+        payout_instructions: str = existing_config.payout_instructions if existing_config is not None else ""
 
-        if make_new_authentication_key or existing_config is None:
-            payout_instructions: str = (await self.standard_wallet.get_new_puzzlehash(in_transaction=True)).hex()
-        else:
-            payout_instructions = existing_config.payout_instructions
+        if len(payout_instructions) == 0:
+            self.log.info(f"Missing payout_instructions for {current_state.launcher_id}. Generating new puzzle hash...")
```
Comment (mariano54) [inline chia/pools/pool_wallet.py:240]: This sounds like an error but it's the normal scenario when syncing up on a new machine. Can we change it to: "New config entry, generating...."

File: chia/pools/pool_wallet.py [hunk 2]
```diff
@@ -229,16 +229,17 @@ async def get_unconfirmed_transactions(self) -> List[TransactionRecord]:
...
+        payout_instructions: str = existing_config.payout_instructions if existing_config is not None else ""
 
-        if make_new_authentication_key or existing_config is None:
-            payout_instructions: str = (await self.standard_wallet.get_new_puzzlehash(in_transaction=True)).hex()
-        else:
-            payout_instructions = existing_config.payout_instructions
+        if len(payout_instructions) == 0:
+            self.log.info("New config entry, generating new puzzle hash...")
+            payout_instructions = (await self.standard_wallet.get_new_puzzlehash(in_transaction=True)).hex()
+            self.log.info(f"Generated payout_instructions puzzle hash: {payout_instructions}")
```
Comment (emlowe) [inline chia/pools/pool_wallet.py:242]: I'd probably prefer this all on one line in the log

Comment (paninaro) [reply chia/pools/pool_wallet.py]: image

Comment (paninaro) [reply chia/pools/pool_wallet.py]: As you wish

**Review by mariano54 — APPROVED** [review]

**Pull request merged**